src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working!");

```

Comment:
The message in the 'doWork' method should indicate the type of machine that is working. Consider rephrasing to include the machine type more clearly. For example: "Bulldozer is starting its work."